rst_files/linear_algebra.rst

```diff
@@ -278,35 +264,29 @@ The expression :math:`\| x - y\|` is thought of as the distance between :math:`x
 Continuing on from the previous example, the inner product and norm can be computed as
 follows
 
+.. code-block:: julia
 
-
+    using LinearAlgebra
 
 
 .. code-block:: julia
 
     dot(x, y)               # Inner product of x and y
 
 
-
 .. code-block:: julia
 
-    sum(x .* y)             # Gives the same result
+    sum(i * j for (i, j) ∈ zip(x, y))             # Gives the same result
 
 
-	
 .. code-block:: julia
 
     norm(x)                 # Norm of x
 
 
-
 .. code-block:: julia
 
-    sqrt(sum(x.^2))         # Gives the same result
-
-
-
-
+    sqrt(sum(abs2, x))         # Gives the same result
```

Comment:
I think that leaving both of these examples in there is probably a good idea.

Reply:
I used `sum(_ -> _^2, x)` and other variants throughout the lectures to introduce a new convenient function.